docs/contracts/contracts/LSP0ERC725Account/LSP0ERC725Account.md

```diff
@@ -0,0 +1,1485 @@
+  uint256[] values,
+  bytes[] datas
+) external payable returns (bytes[]);
+```
+
+_Calling multiple addresses `targets` using `operationsType`, transferring `values` wei and data: `datas`. _
+
+Batch executor function that behaves the same as [`execute`](#execute) but allowing multiple operations in the same transaction.
+
+<blockquote>
+
+**Requirements:**
+
+- The length of the parameters provided MUST be equal
+- If a `value` is provided, the contract MUST have at least this amount in its balance to execute successfully.
+- If the operation type is `STATICCALL` or `DELEGATECALL`, `value` SHOULD be 0.
+- `target` SHOULD be `address(0)` when deploying a contract.
+- MUST pass when called by the owner or by an authorised address that pass the verification check performed on the owner accordinng to [LSP-20-CallVerification] specification.
+
+</blockquote>
+
+<blockquote>
+
+**Emitted events:**
+
+- [`Executed`](#executed) event, when a call is executed under `operationType` 0, 3 and 4 (each iteration)
+- [`ContractCreated`](#contractcreated) event, when a contract is created under `operationType` 1 and 2 (each iteration)
+- [`ValueReceived`](#valuereceived) event when receiving native tokens.
+
+</blockquote>
+
+#### Parameters
+
+| Name             |    Type     | Description                                                                                                     |
+| ---------------- | :---------: | --------------------------------------------------------------------------------------------------------------- |
+| `operationsType` | `uint256[]` | The list of operations type used: `CALL = 0`; `CREATE = 1`; `CREATE2 = 2`; `STATICCALL = 3`; `DELEGATECALL = 4` |
+| `targets`        | `address[]` | The list of addresses to call. `targets` will be unused if a contract is created (operation types 1 and 2).     |
+| `values`         | `uint256[]` | The list of native token amounts to transfer (in Wei).                                                          |
+| `datas`          |  `bytes[]`  | The list of calldata, or the creation bytecode of the contract to deploy if `operationType` is `1` or `2`.      |
+
+#### Returns
+
+| Name |   Type    | Description |
+| ---- | :-------: | ----------- |
+| `0`  | `bytes[]` | -           |
+
+### getData
+
+:::note Links
+
+- Specification details in [**LSP-0-ERC725Account**](https://github.com/lukso-network/lips/tree/main/LSPs/LSP-0-ERC725Account.md#getdata)
+- Solidity implementation in [**LSP0ERC725Account**](https://github.com/lukso-network/lsp-smart-contracts/blob/develop/contracts/LSP0ERC725Account/LSP0ERC725Account.sol)
+- Function signature: `getData(bytes32)`
+- Function selector: `0x54f6127f`
+
+:::
+
+```solidity
+function getData(bytes32 dataKey) external view returns (bytes dataValue);
+```
+
+_Reading the ERC725Y storage for data key `dataKey` returned the following value: `dataValue`._
+
+Get in the ERC725Y storage the bytes data stored at a specific data key `dataKey`.
```

Comment:
```suggestion
Get from the ERC725Y storage the bytes data stored at a specific data key `dataKey`.
```

Reply:
This should be described on the `IERC725X` interface as we inherit it here.